### DISCUSSION TITLE
Creating rule from event

### DISCUSSION THREAD

**Discussion by tavoda** [evilsocket/opensnitch#843] [Q&A]
Am I doing something wrong but I can't create rule from row in Events tab. Is it missing feature or some issue?

**Comment by gustavo-iniguez-goya** [comment]
That functionality is not supported. What are you trying to do?

**Reply by tavoda** [reply]
For example I allow Chrome access everywhere, than I see in Events log some site which I would like to deny (tracking request). I would like to click on this row and just deny access to this host.

**Reply by gustavo-iniguez-goya** [reply]
ok. What options would you expect? something like:

right click on  a row ->:
 - New rule
   * From this executable
   * To this host
   * To this port
   * Custom

**Reply by tavoda** [reply]
Just new rule dialog with filled in values for app, host, port, ... and I can select checkboxes next to item to apply specific condition.
